scheduler.go

```diff
@@ -0,0 +1,163 @@
+package main
+
+import (
+	"database/sql"
+	"errors"
+	"strconv"
+	"time"
+)
+
+type repeatSettings struct {
+	key      string
+	interval int
+}
+
+type scheduler struct {
+	db *sql.DB
+}
+
+type Task struct {
+	Id      string `json:"id"`
+	Date    string `json:"date"`    // "20240201",
+	Title   string `json:"title"`   // "Подвести итог",
+	Comment string `json:"comment"` // "Мой комментарий",
+	Repeat  string `json:"repeat"`  // "d 5"
+}
+
+func NewScheduler(db *sql.DB) scheduler {
+	return scheduler{db}
+}
+
+func (s scheduler) addTask(t Task) (int64, error) {
+	res, err := s.db.Exec("INSERT INTO scheduler (date, title, comment, repeat) VALUES (:date, :title, :comment, :repeat)",
+		sql.Named("date", t.Date),
+		sql.Named("title", t.Title),
+		sql.Named("comment", t.Comment),
+		sql.Named("repeat", t.Repeat),
+	)
+	if err != nil {
+		return 0, err
```

Comment:
На будущее лучше ошибки оборачивать, так по логам можно будет быстрее разобраться, где была ошибки
```suggestion
		return 0, fmt.Errorf("error while inserting task %+v: %w", t, err)
```